tools/functional-tester/etcd-runner/command/global.go

```diff
@@ -23,24 +24,26 @@ import (
 	"github.com/coreos/etcd/clientv3"
 
 	"github.com/spf13/cobra"
+	"golang.org/x/time/rate"
 )
 
 var (
-	rounds                 int           // total number of rounds the operation needs to be performed
 	totalClientConnections int           // total number of client connections to be made with server
 	noOfPrefixes           int           // total number of prefixes which will be watched upon
 	watchPerPrefix         int           // number of watchers per prefix
-	reqRate                int           // put request per second
+	prefix                 string        // prefix append to keys in watcher
 	totalKeys              int           // total number of keys for operation
 	runningTime            time.Duration // time for which operation should be performed
+	ttl                    int           // lease ttl in lease renewer
```

Comment:
this should be `leaseTTL` in `lease_renewer_command.go` if it's only being used by one command